organization.rst

```diff
@@ -0,0 +1,278 @@
+Organization
```

Comment:
This whole file looks redundant and also cannot be in the main directory anyway. Please remove it.